spec/middlewares/time_traveler_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+describe TimeTraveler do
+  context "when called with a POST request" do
```

Comment:
We are not making post requests in those tests. I would axe the most outside context, it's not like there is any other context in this test file.

Reply:
Fixed in [66d81bb](66d81bb)